### PR TITLE
Add eval fid

### PR DESCRIPTION
**Pull request by lingwei-gu**
added evaluator for fid result

### CODE REVIEW

File: docs/experiments-fid-reader.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+Run nvidia-smi to make sure gpu is set up.
+```bash
+pip install pyserini
+export JAVA_HOME="/usr/lib/jvm/java-11-openjdk-amd64"
+git clone https://github.com/castorini/pygaggle.git 
+cd pygaggle
+pip install --editable .
+pip install -r requirements.txt
+gsutil cp gs://ron-random/runs/run.encoded.dkrr.test.json data/
```
Comment (ronakice) [inline docs/experiments-fid-reader.md:18]: remove this.

File: docs/experiments-fid-reader.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+bash get-model.sh -m nq_reader_base
+pip install -r requirements.txt
+!pip install transformers==4.10.0
+```
+
+After downloading the model, don't forget to change the model name being used in pygaggle/qa/fid_reader.py
+
+Then we run the inference and evaluation in the TOP level directory of Pygaggle:
+```bash
+$ python -um pygaggle.run.evaluate_fid_ranker --task wikipedia --retriever score --reader fid \
```
Comment (ronakice) [inline docs/experiments-fid-reader.md:35]: where is the nq_reader_base passed to the script, I'm confused?

Comment (lingwei-gu) [reply docs/experiments-fid-reader.md:35]: nq_reader_base needs to be changed in fid_reader.py

Comment (lingwei-gu) [reply docs/experiments-fid-reader.md:35]: I'll update that too

**Review by ronakice — APPROVED** [review]
LGTM!

**Pull request merged**